### ISSUE TITLE
How to execute the 'new" function to create the strategy object?

### ISSUE THREAD

**Issue by Jarekkkkk** [kunalabs-io/sui-smart-contracts#8]
Hello, 

i noticed a **new** function in some contracts, like **scallop_whusdte.move**, that requires creating the shared strategy object . The function has been assigned the **entry modifier** and **friend visibility** but doesn't introduced with any friend package declaration, may i ask what the purpose of this pattern is? 

example code: https://github.com/kunalabs-io/sui-smart-contracts/blob/8151c3690a2f60276b4d38b02dfc53650ae898ee/kai-finance/sources/scallop_whusdte.move#L84C30-L84C33

**Comment by kklas** [comment]
Indeed, `friend` visibility doesn't achieve anything in this case and can be removed. I had it like this originally because I was planning to create strategies from other modules within the package but changed the approach since. The funciton is private for better upgradeability.
